base/math.jl

```diff
@@ -986,6 +986,11 @@ This function computes a floating point representation of the modulus after divi
 numerically exact `2π`, and is therefore not exactly the same as `mod(x,2π)`, which would
 compute the modulus of `x` relative to division by the floating-point number `2π`.
 
+!!! note
+    Depending on how your machine represents floats, `2π` may be less than the numerically
```

Comment:
I would say something like "Depending on the format of the input value, the closest representable value to 2π, may be less than 2π. For example, the expression `mod2pi(2π)` will not return zero, as the intermediate `2*π` defaults to `Float64`, and we have `2*Float64(π) < 2*big(π)`.

Reply:
Updated